code/game/machinery/cryo.dm

```diff
@@ -579,6 +579,8 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
 	if(M.health > -100 && (M.health < 0 || M.sleeping))
 		to_chat(M, "<span class='bnotice'>You feel a cold liquid surround you. Your skin starts to freeze up.</span>")
 	occupant = M
+	for(var/obj/item/I in M.held_items)
```

Comment:
Did you test this? I believe `held_items` is an associative list, in which case this may be looping through the keys rather than the items.